packages/smart/funcs.go

```diff
@@ -788,6 +796,34 @@ func ColumnCondition(sc *SmartContract, tableName, name, coltype, permissions, i
 	return sc.AccessTable(tblName, "new_column")
 }
 
+// RowConditions checks conditions for table row by id
+func RowConditions(sc *SmartContract, tblname string, id int64) error {
```

Comment:
Should be CheckRowConfitions(), I think

Reply:
May be. But this name was chosen in the style of other TableConditions, ColumnConditions.